# Names and values match TelloPy
uint8 FLIP_FORWARD=0
uint8 FLIP_LEFT=1
uint8 FLIP_BACK=2
uint8 FLIP_RIGHT=3
uint8 FLIP_FORWARDLEFT=4
uint8 FLIP_BACKLEFT=5
uint8 FLIP_BACKRIGHT=6
uint8 FLIP_FORWARDRIGHT=7
uint8 flip_command
